docs/graphql/core/schema/custom-functions.rst

```diff
@@ -551,6 +552,26 @@ following example.
 
    The specified session argument will not be included in the ``<function-name>_args`` input object in the GraphQL schema.
 
+
+Using functions as mutations
+****************************
+
+You can also use the :ref:`track_function_v2 <track_function_v2>` API to track
+functions as mutations. In this case the SQL function *must* be marked
+``VOLATILE`` (this is the default if volatility isn't specified; see 
+`the postgres docs <https://www.postgresql.org/docs/current/xfunc-volatility.html>`).
```

Comment:
Something wrong with the link here:

![image](https://user-images.githubusercontent.com/5722022/97959416-9fa7f900-1daf-11eb-91b6-4707c960d33b.png)